docs/docs/articles/testkube-licensing-FAQ.md

```diff
@@ -0,0 +1,65 @@
+# Testkube Licensing FAQ
+
+Testkube's software licensing is designed to be transparent and to support both open source and commercial use cases. This document aims to address common questions related to our licensing model.
+
+## Licenses
+
+Testkube software is distributed under two primary licenses:
+- **MIT License (MIT)**: A permissive open-source license that allows for broad freedom in usage and modification.
+- **Testkube Community License (TCL)**: A custom license designed to protect the Testkube community and ecosystem, covering specific advanced features.
+
+## Testkube Core
+
+Testkube Core is free to use. Most core features are licensed under the MIT license, but some core features are subject to the TCL.
+
+## Testkube Pro
+
+Testkube Pro features require a paid license from Testkube (see [pricing](https://testkube.io/pricing)) and are licensed under the Testkube Community License.
+
+:::note
+You can find any feature's license by checking the code's file header in the Testkube repository.
+:::
+
+### What is the TCL License?
+
+The Testkube Community License (TCL) is a custom license created by Testkube to cover certain aspects of the Testkube software. It was inspired by the [CockroachDB Community License](https://www.cockroachlabs.com/docs/stable/licensing-faqs#ccl) and designed to ensure that advanced features and proprietary extensions remain available and maintained for the community while allowing Testkube to sustain its development through commercial offerings.
+
+### Why does Testkube have a dual-licensing scheme with MIT / TCL?
```

Comment:
Proposed new language:

Why does Testkube have a dual licensing scheme with MIT and TCL?

Reply:
@irh123 👍